DDS: Cofense Triage Readme Update (#21832)
DataDog/integrations-core: cofense_triage/README.md

   | Client Secret | The Client Secret of your Cofense Triage account. |
2. Click **Save** to save your settings.

#### Whitelist Datadog IP Addresses

> 1. Use an API platform such as Postman, or curl to make a GET request to the [Datadog API endpoint][3].

Author: Done

> 2. Once you receive the response, locate the **webhooks** section in the JSON. It will look something like this:

Reviewer: ```suggestion
2. After you receive the response, locate the **webhooks** section in the JSON. It looks something like this:
```

Author: Done
